src/coreComponents/physicsSolvers/fluidFlow/ReactiveCompositionalMultiphaseOBL.cpp

```diff
@@ -372,7 +372,7 @@ real64 ReactiveCompositionalMultiphaseOBL::scalingForSystemSolution( DomainParti
     return 1.0;
   }
 
-  real64 constexpr eps = ReactiveCompositionalMultiphaseOBLKernels::minValueForDivision;
+  real64 constexpr eps = reactiveCompositionalMultiphaseOBLKernels::minValueForDivision;
```

Comment:
Since you have the `using` up there, do you need the namespace here too?

Reply:
Yes, you are right, I removed the namespace here